site/blog/updates/why-sql-for-infrastructure.mdx

```diff
@@ -168,6 +168,18 @@ resource "aws_instance" "login" {
 }
 ```
 
+This example still requires the following lines from the imperative example to get the public IP address of the new EC2 instance and log into it:
+
+```bash
+PUBLIC_IP=$(aws ec2 describe-instances \
```

Comment:
Did you consider making this an [HCL output](https://developer.hashicorp.com/terraform/language/values/outputs) and [accessing it via bash](https://stackoverflow.com/a/69260021/2594912)?

Reply:
So, the terraform code was the longest of the three, so I thought adding that and then still having a bash script afterwards would unfairly paint Terraform even worse, but I suppose an argument could be made that I'm unfairly painting Terraform as being unable to get that public IP address to use in the bash script.

I *know* the Terraform code is going to draw rotten tomatoes at us either way, but let me think about which is "worse." (And go with that one? :zany_face:)